src/css/components/_modal-dialog.scss

```diff
@@ -18,6 +18,8 @@
 .vjs-modal-dialog .vjs-modal-dialog-content {
   @extend %fill-parent;
 
+  @include user-select(auto);
```

Comment:
We're [disabling user-selection on the player div](https://github.com/videojs/video.js/blob/master/src/css/components/_layout.scss#L24) so that it doesn't get highlighted if you're dragging the progress bar. For whatever reason, even though, it's not supposed to be inherited, it applies on this element.